src/connectedmachine/report.md

```diff
@@ -0,0 +1,183 @@
+# Azure CLI Module Creation Report
```

Comment:
what's this for? It should be auto-gen code report, no need to checkin

Reply:
No idea it was there as a part of the generation